add link to governance doc repo (#744)
open-life-science/open-life-science.github.io: governance.md

{% assign people = site.data.people %}

# Governance Committee

> You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance).

Reviewer: Why the first sentence has been removed?

Reviewer 2: Fixed now. :)

> You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance).

Reviewer 2: ```suggestion
The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS. You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance).
```

> The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS. You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance)

Reviewer 2: ```suggestion
You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance).
```

Reviewer 2: If Bérénice has to remind us to use variable_url one more time, someone might need a beating.

It looks good, Yo! Sorry I couldn't resist the urge to punctuate. :)

> The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS. You can also [read the OLS governance guiding documents](https://github.com/open-life-science/ols-governance)

Reviewer: ```suggestion
The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS. You can also [read the OLS governance guiding documents]({{ site.github.owner_url }}/ols-governance)
```

Author: thanks!! 🌹  I wondered if I was missing a nifty variable like that

Reviewer 3: yeah. Old habits die hard. Variablesss!
